tests/cdc-bank/bank.go

```diff
@@ -192,9 +216,14 @@ func (c *client) Start(ctx context.Context, cfg interface{}, clientNodes []clust
 			defer wg.Done()
 			for {
 				select {
-				case <-ctx.Done():
-					return
-				default:
+				case tblName := <-tblChan:
+					waitCtx, waitCancel := context.WithTimeout(ctx, 2*time.Minute)
+					waitTable(waitCtx, downstream, tblName)
+					waitCancel()
+					log.Infof("ddl synced")
```

Comment:
When the validator receives notification from `tblChan`, it seems more DMLs after this DDL could have been executed in upstream?